x-pack/legacy/plugins/lens/public/editor_frame_plugin/embeddable/embeddable_factory.ts

```diff
@@ -9,7 +9,7 @@ import { Chrome } from 'ui/chrome';
 
 import { capabilities } from 'ui/capabilities';
 import { i18n } from '@kbn/i18n';
-import { IndexPatterns, IndexPattern } from 'src/legacy/core_plugins/data/public';
+import { IndexPatternsContract, IndexPattern } from '../../../../../../../src/plugins/data/public';
```

Comment:
Why can't this be called `IndexPatternsService` rather than `Contract`?

Reply:
It is a popular convention used for the types that can be used to pass the services around (`SavedObjectsClientContract`, `HttpServiceContract`,etc)